### PR TITLE
CMR-9424: Improve code coverage for common-lib to 65%

### PR DESCRIPTION
**Pull request by TylerHeald1**
Added some simple unit tests to increase coverage past the required minimum

### CODE REVIEW

File: common-lib/test/cmr/common/test/util.clj [hunk 1]
```diff
@@ -44,20 +60,42 @@
...
-      2 (+ 1 1)
-      "A more complicated test"
-      4 (* 2 2))))
+     [x y] (= x y)
+     "The most basic case with 1"
+     2 (+ 1 1)
+     "A more complicated test"
+     4 (* 2 2)))
+  (testing "Empty call"
+    (util/are2
```
Comment (jceaser) [inline common-lib/test/cmr/common/test/util.clj:69]: are2 is not to be used, please move to are3

Comment (TylerHeald1) [reply common-lib/test/cmr/common/test/util.clj]: Removed the new are2 test that I added. Replaced with a new test for a different util function

File: common-lib/test/cmr/common/test/api/context.clj [hunk 2]
```diff
@@ -0,0 +1,26 @@
+(ns cmr.common.test.api.context
+  "This tests capabilities of the API context utilities."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :as util]
+   [cmr.common.api.context :as context]))
+
+(deftest request-context-test
```
Comment (jmaeng72) [inline common-lib/test/cmr/common/test/api/context.clj:8]: Can you write a comment or something to show what use cases these are testing?

**Comment by jmaeng72** [comment]
I'm seeing on codecov that common-lib on master is already at 64.71% coverage on codecov. How can I see on codecov that common-lib has increased to 65% from your branch?

File: common-lib/test/cmr/common/test/api/context.clj (hunk 2 above)
Comment (TylerHeald1) [reply common-lib/test/cmr/common/test/api/context.clj:8]: Added 'testing' map around tests to give some info

**Review by jmaeng72 — APPROVED** [review]
LGTM

**Review by zimzoom — APPROVED** [review]

**Pull request merged**